src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    public static final int OPERATION_TYPE = 0;
+    public static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+        int supplySum = 0;
+        int buySum = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile));
+                BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFile))) {
```

Comment:
Use try-with-resources to ensure that both `BufferedReader` and `BufferedWriter` are closed properly. It's good to see you've used it here, but make sure the resources are closed even if an exception occurs before the try block ends.